src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/CollectionRulePipelineTests.cs

```diff
@@ -160,6 +160,206 @@ public Task CollectionRulePipeline_EventCounterTriggerTest(TargetFrameworkMonike
                 });
         }
 
+        /// <summary>
+        /// Test that the pipeline works with the EventMeter trigger (gauge instrument).
+        /// </summary>
+        [Theory]
+        [InlineData(TargetFrameworkMoniker.Net70)]
```

Comment:
```suggestion
        [InlineData(TargetFrameworkMoniker.Net80)]
```

If we are only going to run these on one TFM, then it needs to be `Net8.0` so that PR will run them. However, this will prevent them from running in VS until we allow building the `net8.0` TFM in VS. This could be done with a `MemberData` attribute that calls a method that determines whether the test is running in VS based on an environment variable and return either `Net70` or `Net80`.

Reply:
I mimicked what the existing TFM logic does with `INCLUDE_NEXT_DOTNET` - let me know if you have something else in mind